fps/src/main/scala/chapter02/exercise2_2.scala

```diff
@@ -0,0 +1,23 @@
+package chapter02
+
+
+object exercise2_2 extends App {
+
+  def simpleSort(a: String, b: String): Boolean = {
+    b == a + a
+  }
+
+  def isSorted[A](as: Array[A], ordered: (A,A) => Boolean): Boolean = {
+    // Make list with tuple of all combinations
+    // check the comparison function per combination
+    // check if all are true
+    as.dropRight(1)
```

Comment:
`as.zip(as.tail)` also works - ie. `zip` returns a zipped list with the same length as the shortest list, ie. dropping the elements of the longer list, tail side